bin/setup_dot1x

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash -e
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+
+sudo rm -rf inst/faucet/faucet_dot1x
+mkdir -p inst/faucet/faucet_dot1x
+
+cp misc/faucet_dot1x.yaml inst/faucet/faucet_dot1x/faucet.yaml
+
+sudo ovs-vsctl del-br br-dot1x || true
+
+docker rm -f faucet_dot1x || true
+docker rm -f daq-ntwk || true
+docker rm -f daq-faux-1 || true
+docker rm -f daq-faux-1 || true
+
+cmd/faux 1 xdhcp 
+cmd/faux 2 xdhcp 
+
+echo "Created faux containers"
+
+docker cp misc/wpasupplicant.conf.freeradius daq-faux-1:/root/wpasupplicant.conf
+
+docker exec daq-faux-1 ip addr flush faux-eth0
+docker exec daq-faux-1 ifconfig faux-eth0 192.168.12.3/16
+docker exec daq-faux-2 ip addr flush faux-eth0
+docker exec daq-faux-2 ifconfig faux-eth0 192.168.12.2/16
+
+echo "set IPs for faux"
+
+docker run -d --privileged --name daq-ntwk --hostname daq-ntwk daq/networking
+echo -e "daq up"
+docker exec  daq-ntwk ip addr flush eth0
+echo -e"ip flushed"
+ip_fr=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d '/' -f1)2
+docker exec  daq-ntwk ifconfig eth0 $ip_fr
+
+
+docker cp misc/clients.conf.freeradius daq-ntwk:/etc/freeradius/3.0/clients.conf
+docker cp misc/authorize.freeradius daq-ntwk:/etc/freeradius/3.0/mods-config/files/authorize
+docker exec daq-ntwk freeradius -X &
+
+echo "freeradius setup"
+
+cmd/faucet faucet_dot1x 6653
+docker exec faucet_dot1x ip addr flush eth0
+ip_fc=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d '/' -f1)1
+docker exec faucet_dot1x ifconfig eth0 $ip_fc
+
+echo  "faucet setup"
+
+pid=$(docker inspect --format="{{ .State.Pid }}" faucet_dot1x)
+sudo ip link add dplane0 type veth peer name dplane-eth0 addr 9a:02:57:1e:8f:2a netns $pid
+sudo ip link set dplane0 up
+docker exec faucet_dot1x ip link set dplane-eth0 up
+
+
+sudo ovs-vsctl --if-exists del-br br-dot1x \
+    -- add-br br-dot1x \
+    -- set bridge br-dot1x other_config:datapath-id=0x250
+
+echo  "Created bridge"
+
+sudo ovs-vsctl set-controller br-dot1x tcp:$ip_fc:6653
+
+
+sudo ovs-vsctl add-port br-dot1x faux-1  -- set interface faux-1 ofport_request=1
+sudo ovs-vsctl add-port br-dot1x faux-2 -- set interface faux-2 ofport_request=2
+sudo ovs-vsctl add-port br-dot1x dplane0 -- set interface dplane0 ofport_request=3
+
+
+#docker exec faucet_dot1x pkill -HUP -f faucet.faucet
```

Comment:
remove dead code, or have a explanation of what it's for.  generally dead code just leads to more problems in the long run.